extensions-contrib/parquet-extensions/src/main/java/io/druid/data/input/parquet/ParquetHadoopInputRowParser.java

```diff
@@ -56,15 +61,50 @@ public ParquetHadoopInputRowParser(
     }
   }
 
+  private static LogicalType _timestampSpecLogicalType = null;
+
+  private LogicalType determineTimestampSpecLogicalType(Schema schema, String timestampSpecFields)
```

Comment:
Why the parameter is called "timestampSpecField**s**"? It is compared with singular fields in the body.